pkg/backend/display/progress.go

```diff
@@ -1229,6 +1229,8 @@ func (display *ProgressDisplay) getStepDoneDescription(step engine.StepEventMeta
 				opText = "imported"
 			case deploy.OpImportReplacement:
 				opText = "imported replacement"
+			case deploy.OpRemovePendingReplace:
+				opText = ""
```

Comment:
I think these two changes are OK -- it seems unimportant to me to signal a purely state-level change to the user, but happy to tweak if this is a bad assumption.